registry/kusama.json

```diff
@@ -118,5 +125,4 @@
          "shortDescription": "Web3 buildooor @ Astar Network & validator",
          "longDescription": "## About bLd\n\nDevops engineer at [Astar Network](https://astar.network) \n\nValidator at [bLd Nodes](https://bldnodes.org) \n\nMember of [DOT Validator Alliance DAO](https://dotvalidators.org) \n\nOrganizer at [ParisDotComm](https://parisdotcomm.org) \n\nCurator for [IBP program](https://ibp.network) \n\n---\n\nVoting constantly with conviction, I believe governance is the affair of everyone and public blockchain is a common good.\nMy votes represent my personal opinions, not in any case the views of any of my organisations.\nI happily explain my votes to anyone reaching and change my vote when I'm proved mistaken.  \n\n## Contact \n\n@[Twitter](https://twitter.com/bLd77) @bLd77 \n\n@[Telegram](https://t.me/bLd759) @bLd759 \n\n@[Github](https://github.com/bLd75) @bLd75",
          "isOrganization": false
-     }
```

Comment:
you should not remove that closing brackets

Reply:
k.